airbyte-integrations/connectors/source-pardot/source_pardot/manifest.yaml

```diff
@@ -0,0 +1,739 @@
+  base_requester:
+    type: HttpRequester
+    url_base: https://pi.pardot.com/api/
+    authenticator:
+      type: OAuthAuthenticator
+      refresh_request_body: {}
+      token_refresh_endpoint: >-
+        https://{{ 'test' if config['is_sandbox'] else 'login'
+        }}.salesforce.com/services/oauth2/token
+      grant_type: refresh_token
+      client_id: '{{ config["client_id"] }}'
+      client_secret: '{{ config["client_secret"] }}'
+      refresh_token: '{{ config["client_refresh_token"] }}'
+
+streams:
+  - $ref: "#/definitions/streams/Campaigns"
+  - $ref: "#/definitions/streams/EmailClicks"
+  - $ref: "#/definitions/streams/ListMembership"
+  - $ref: "#/definitions/streams/Lists"
+  - $ref: "#/definitions/streams/ProspectAccounts"
+  - $ref: "#/definitions/streams/Prospects"
+  - $ref: "#/definitions/streams/Users"
+  - $ref: "#/definitions/streams/VisitorActivities"
+  - $ref: "#/definitions/streams/Visitors"
+  - $ref: "#/definitions/streams/Visits"
+  # - $ref: "#/definitions/streams/Opportunities"
+
+spec:
+  type: Spec
+  connection_specification:
+    "$schema": http://json-schema.org/draft-07/schema#
+    title: Pardot Spec
+    type: object
+    required:
+      - pardot_business_unit_id
+      - client_id
+      - client_secret
+      - refresh_token
+    additionalProperties: false
+    properties:
+      pardot_business_unit_id:
+        description:
+          Pardot Business ID, can be found at Setup > Pardot > Pardot Account
+          Setup
+        type: string
+      client_id:
+        description: The Consumer Key that can be found when viewing your app in Salesforce
+        type: string
+        airbyte_secret: true
+      client_secret:
+        description:
+          The Consumer Secret that can be found when viewing your app in
+          Salesforce
+        type: string
+        airbyte_secret: true
+      refresh_token:
+        description:
+          Salesforce Refresh Token used for Airbyte to access your Salesforce
+          account. If you don't know what this is, follow this <a href="https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b">guide</a>
+          to retrieve it.
+        type: string
+        airbyte_secret: true
+      start_date:
+        description:
+          UTC date and time in the format 2017-01-25T00:00:00Z. Any data
+          before this date will not be replicated. Leave blank to skip this filter
+        type: string
+        pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+        default:
+        examples:
+          - "2021-07-25T00:00:00Z"
+      is_sandbox:
+        description:
+          Whether or not the the app is in a Salesforce sandbox. If you do
+          not know what this, assume it is false.
+        type: boolean
+        default: false
+
+schemas:
+  Campaigns:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties:
```

Comment:
Sir this is a yaml file, it looks like these are just copy and pasted from JSON, I _think_ they won't work.